Runners/ConsoleApp/Program.cs

```diff
@@ -23,14 +26,30 @@ static void Main()
         void Main(IMessageSender messageSender)
         {
             var pluginExecutor = new PluginExecutor(messageSender, new MemoryDal(), new PluginsMenu(), new PluginsManager());
-
+            Console.WriteLine("Hi! I am Your amazing bot and I will be happy to be of service to you on several topics.");
+            Console.WriteLine("Please type the number of the plugin you would like to help, or type help to get the list of options.");
+            Console.WriteLine("At any point you can type MAIN or HOME to return to the main menu.");
+            
             while (true)
             {
+                var PluginId= pluginExecutor.getPluginId(" ");
+                PluginId = (PluginId!= null && PluginId!="") ? "\\" + PluginId : "";
+                Console.Write("main{0}> ", PluginId);
                 var msg = Console.ReadLine();
                 pluginExecutor.Run(msg, " ");
             }
         }
 
+        private void OutputTextWithDelay(string txt)
```

Comment:
This function is not in use, please rewove

Reply:
OK, I forgot to delete